main.css

```diff
@@ -0,0 +1,743 @@
+}
+
+.margin1 {
+    margin: 1rem;
+}
+
+.margin2 {
+    margin: 2rem;
+}
+
+.margin3 {
+    margin: 3rem;
+}
+
+.margin4 {
+    margin: 3rem;
+}
+
+.ml-1 {
+
+    margin-left: 1rem;
+}
+
+.ml-2 {
+
+    margin-left: 2rem;
+}
+
+.ml-3 {
+
+    margin-left: 3rem;
+}
+
+.ml-5 {
+
+    margin-left: 5rem;
+}
+
+
+.mr-1 {
+
+    margin-right: 1rem;
+}
+
+.mr-2 {
+
+    margin-right: 2rem;
+}
+
+.mr-3 {
+
+    margin-right: 3rem;
+}
+
+.mt-1 {
+
+    margin-top: 1rem;
+}
+
+.mt-2 {
+
+    margin-top: 2rem;
+}
+
+.mt-3 {
+
+    margin-top: 3rem;
+}
+
+.mb-1 {
+
+    margin-bottom: 1rem;
+}
+
+.mb-2 {
+
+    margin-bottom: 2rem;
+}
+
+.mb-3 {
+
+    margin-bottom: 3rem;
+}
+
+.padding0 {
+    padding: 0.5rem;
+}
+
+.padding1 {
+    padding: 1rem;
+}
+
+.padding2 {
+    padding: 2rem;
+}
+
+.padding3 {
+    padding: 3rem;
+}
+
+.padding4 {
+    padding: 4rem;
+}
+
+
+.pl-1 {
+
+    padding-left: 1rem;
+}
+
+.pl-2 {
+
+    padding-left: 2rem;
+}
+
+.pl-3 {
+
+    padding-left: 3rem;
+}
+
+
+.pr-1 {
+
+    padding-right: 1rem;
+}
+
+.pr-2 {
+
+    padding-right: 2rem;
+}
+
+.pr-3 {
+
+    padding-right: 3rem;
+}
+
+.pt-1 {
+
+    padding-top: 1rem;
+}
+
+.pt-2 {
+
+    padding-top: 2rem;
+}
+
+.pt-3 {
+
+    padding-top: 3rem;
+}
+
+.pb-1 {
+
+    padding-bottom: 1rem;
+}
+
+.pb-2 {
+
+    padding-bottom: 2rem;
+}
+
+.pb-3 {
+
+    padding-bottom: 3rem;
+}
+
+.text-align-center {
+    text-align: center;
+}
+
+.text-align-left {
+    text-align: left;
+}
+
+.text-align-right {
+    text-align: right;
+}
+
+.heading-bold {
+    font-size: 30px;
+    color: black;
+}
+
+.min-font-size {
+    font-size: 20px;
+}
+
+.text-shadow-effect {
+    text-shadow: 2px 2px 5px grey;
+}
+
+.icon-height {
+    height: 40px;
+    width: 40px;
+}
+
+.avatar>img {
+    width: 10%;
+    height: 100%;
+    border: 1px solid black;
+}
+
+.rounded>img {
+    border-radius: 50%;
+}
+
+.very-small>img {
+    width: 2.5rem;
+    height: 2.5rem;
+}
+
+.d-inline {
+    display: inline;
+}
+
+.medium-small>img {
+    width: 3rem;
+    height: 3rem;
+}
+
+.medium>img {
+    width: 3.5rem;
+    height: 3.5rem;
+}
+
+.large>img {
+    width: 4rem;
+    height: 4rem;
+}
+
+.extra-large>img {
+    width: 4.5rem;
+    height: 4.5rem;
+}
+
+.dot-online {
+    height: 15px;
+    width: 15px;
+    background-color: green;
+    border-radius: 50%;
+    display: inline-block;
+    position: absolute;
+
+}
+
+.dot-offline {
+    height: 15px;
+    width: 15px;
+    background-color: red;
+    border-radius: 50%;
+    display: inline-block;
+    position: absolute;
+    right: 28.5rem;
+
+}
+
+.fontWeight {
+    font-weight: bolder;
+
+}
+
+.alert {
+    padding: 1rem;
+    background-color: transparent;
+    color: white;
+}
+
+.alert-danger {
+    padding: 1rem;
+    background-color: #f44336;
+    color: white;
+}
+
+.alert-success {
+    padding: 1rem;
+    background-color: green;
+    color: white;
+}
+
+.alert-warning {
+    padding: 1rem;
+    background-color: yellow;
+    color: white;
```

Comment:
Since, the white color isn't visible on yellow. The color could be replaced to black or any other color visible on yellow.